src/Tizen.TV.UIControls.Forms/GridViewItemFocusedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Xamarin.Forms;
+
+namespace Tizen.TV.UIControls.Forms
+{
+    /// <summary>
+    /// Arguments for the event that is raised when one item of GridView has received focus. 
+    /// </summary>
+    public class GridViewFocusedEventArgs : FocusEventArgs
+    {
+        /// <summary>
+        /// The data for focused item.
+        /// </summary>
+        public object Data { get; }
```

Comment:
`Item` is more proper because, this object is a contained in `ItemsSource`
```suggestion
        public object Item { get; }
```

Reply:
I fixed it. Thank you.